src/userdev/java/net/minecraftforge/gradle/userdev/DependencyManagementExtension.java

```diff
@@ -63,6 +65,11 @@ public Dependency deobf(Object dependency, Closure<?> configure) {
         return remapper.remap(baseDependency);
     }
 
+    @SuppressWarnings("unused")
```

Comment:
No need for the suppress unused on public methods...

Reply:
Then why did you/someone else do it on the extension API function? Just trying to stick to what was already there..

Reply:
Honestly, no idea IDEA users like to throw suppress warnings all over the place that themselves end up just causing warnings in other IDEs. There is no reason to have a suppress here.

Reply:
Removed redundant supressions in latest commit